tests/unit/Storefront/Page/Account/Order/AccountOrderEditPageLoaderTest.php

```diff
@@ -204,6 +204,7 @@ public function testCartInCartService(): void
             ->willReturn($orderContext);
 
         $request = new Request();
```

Comment:
do you even need that, if you create a new request below?

Reply:
Nope, I explicitly want to test, that the `onlyAvailable` flag is set in the page loader. 
This is only the request I expect.
Thats why I pass an empty request to the page loader.